src/statscollector/DefaultStatsCollector.ts

```diff
@@ -453,44 +449,22 @@ export default class DefaultStatsCollector implements StatsCollector {
     if (!this.audioVideoController.rtcPeerConnection) {
       return;
     }
+
     const rawMetricReports: RawMetricReport[] = [];
-    if (!this.browserBehavior.requiresPromiseBasedWebRTCGetStats()) {
-      // @ts-ignore
-      this.audioVideoController.rtcPeerConnection.getStats(
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (res: any) => {
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          res.result().forEach((report: any) => {
-            const item: { [name: string]: StatsReportItem } = {};
-            report.names().forEach((name: string) => {
-              item[name] = report.stat(name);
-            });
-            item.id = report.id;
-            item.type = report.type;
-            item.timestamp = report.timestamp;
-            rawMetricReports.push(item);
-          });
-          this.handleRawMetricReports(rawMetricReports);
-        },
-        // @ts-ignore
-        (error: Error) => {
-          this.logger.error(error.message);
-        }
-      );
-    } else {
-      // @ts-ignore
-      this.audioVideoController.rtcPeerConnection
-        .getStats()
-        .then((report: RTCStatsReport) => {
-          report.forEach((item: StatsReportItem) => {
-            rawMetricReports.push(item);
-          });
-          this.handleRawMetricReports(rawMetricReports);
-        })
-        .catch((error: Error) => {
-          this.logger.error(error.message);
+
+    // @ts-ignore
+    this.audioVideoController.rtcPeerConnection
```

Comment:
If we decide to keep this API https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideocontroller/DefaultAudioVideoController.ts#L219, then you can just use it instead of access the stat like this.

Reply:
nit: Also can we use async and await here?

Reply:
> If we decide to keep this API https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideocontroller/DefaultAudioVideoController.ts#L219, then you can just use it instead of access the stat like this.

As discussed offline we want to deprecate `audioVideo.getRTCPeerConnectionStats` in the future, so I will not use it here.